samples/net/common/sample_app_setup.c

```diff
@@ -0,0 +1,222 @@
+/* sample_app_setup.c */
+
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if 1
```

Comment:
@jukkar : Ack, so do we need "#if 1" line? If you say "yes", I'm only happy, I love these, and will include them in my patches ;-).